### ISSUE TITLE
Footer external link icons not accessible

### ISSUE THREAD

**Issue by sukhrajghuman** [govau/dta-website-rebuild#591]
- [x] I’ve read and understood the [Contributing guidelines](./CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](./CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

From visible inspection the external links do not meet colour contrast requirements. The [body component](https://designsystem.gov.au/components/body/) has built in styling for external links. I would add the `external` value to the `rel` attribute of the footer links and remove the `<span class="ext">` so you have something that looks like this:

![image](https://user-images.githubusercontent.com/20184809/48754563-3627a080-ece6-11e8-9151-d481b670fb3a.png)

instead of this:

![image](https://user-images.githubusercontent.com/20184809/48754889-71769f00-ece7-11e8-8afa-ed770a445121.png)

This will also make it easier if you change your colour theme in future, since those updates will be applied to these links as well

**Comment by imperator99** [comment]
Thanks @sukhrajghuman for pointing this out. Looking at it more closely I've decided to remove the icons altogether from the footer, it looks silly having the icons there when they are clearly external. See https://github.com/govau/dta-website-rebuild/pull/604.
